codespan-reporting/src/term/views.rs

```diff
@@ -135,6 +135,34 @@ where
                 }
             };
 
+            let lines_before = usize::min(start_line_index, self.config.before_label_lines);
+            for offset in (1..lines_before + 1).rev() {
+                let range = files.line_range(label.file_id, start_line_index - offset);
+                if let Ok(range) = range {
+                    let line = labeled_file.get_or_insert_line(
+                        start_line_index - offset,
+                        range,
+                        start_line_number - offset,
+                    );
+                    line.must_render = true;
+                } else {
+                    break;
+                }
+            }
+            for offset in 1..lines_before + 1 {
+                let range = files.line_range(label.file_id, start_line_index + offset);
```

Comment:
I think this should start after the end of this label, i.e. from `end_line_lindex`.

Reply:
Fixed this 👍🏻